vpr/src/place/timing_place.cpp

```diff
@@ -14,71 +19,49 @@
 
 #include "timing_info.h"
 
-//Use an incremental approach to updaing criticalities?
-constexpr bool INCR_UPDATE_CRITICALITIES = true;
-
-/**************************************/
+///@brief Use an incremental approach to updating criticalities and setup slacks?
```

Comment:
Explain what this means.  Does it control incremental update_timing calls, or recomputing only the criticalities and setup slacks for connections whose slack changed in the last timing analysis, or both?

Reply:
I think it is safe to remove these flags. We can use incremental updates whenever it's possible now.